cluster/common.sh

```diff
@@ -603,6 +603,7 @@ KUBELET_TOKEN: $(yaml-quote ${KUBELET_TOKEN:-})
 KUBE_PROXY_TOKEN: $(yaml-quote ${KUBE_PROXY_TOKEN:-})
 ADMISSION_CONTROL: $(yaml-quote ${ADMISSION_CONTROL:-})
 MASTER_IP_RANGE: $(yaml-quote ${MASTER_IP_RANGE})
+MASTER_RESERVED_IP: $(yaml-quote ${MASTER_RESERVED_IP})
```

Comment:
Here is the summary how this PR will affect pricing.
- Current state: (1) **single-master cluster** with one external IP address attached to the master, kubelets communicates through internal IP, no additional cost.
- Proposed solution:
   - (2) **single-master cluster**: one external IP address attached to the master, kubelets communicates through external IP; additional cost $0.01 for GB (communication through external IP).
   - (3) **multi-master cluster**: one external load balancer, kubelets communicates through external load balancer; additional cost: each master replica: $0.025 / hour (first five replicas) or $0.010 / hour (next replicas) + GB of data processed: $0.008 / GB.